CHANGELOG.md

```diff
@@ -1,119 +1,54 @@
-# PWA Studio Release 12.5.0
+# PWA Studio Release 12.6.0
 
 **NOTE:**
-_This changelog only contains release notes for PWA Studio and Venia 12.5.0_.
+_This changelog only contains release notes for PWA Studio and Venia 12.6.0_.
 _For older release notes, see_ [PWA Studio releases][].
 
 ## Highlights
 
```

Comment:
Did we want to include mention removal of QuoteGraphQlPwa module, PWA-2477?

Reply:
Done